cexio/__init__.py

```diff
@@ -168,6 +168,24 @@ def order_book(self, depth=1, market='BTC/USD'):
     def trade_history(self, since=1, market='BTC/USD'):
         return self.api_call('trade_history', None, market + '/?since=' + str(since))
 
-    def __post(self, url, param):
+    def __post(self, url, param): #this line changed added proxies, add after param to use socks procy (, proxies="socks5://175.155.68.86:38801")
```

Comment:
Don't understand why you didn't edit this line. 🤔 
I think you should add _proxies_ as optional paramater, like; 
```py
def __post(self, url, param, proxies={})
...
result = requests.post(url, data=param, headers={ 'User-agent': 'bot-cex.io-' + self.username }, proxies=proxies).json()
````
Something like this, maybe ?

Reply:
Sure, I am new to python and wasn't sure if that would work. just tested with proxies="xyz address" and it worked for me, I put that in the code as an option.

Reply:
Maybe we shouldn't add proxy support yet. I don't think it is necessary tbh.